src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -224,6 +224,9 @@ consistent "has a" relationship is tracked by this ticket: https://broadworkbenc
                                                 memberPolicies: Option[Set[PolicyIdAndEmail]] = Option(Set.empty)) {
   def getDescendantPermissions: Set[AccessPolicyDescendantPermissions] = descendantPermissions.getOrElse(Set.empty)
 }
+// AccessPolicyWithMembership is practically the same as AccessPolicyResponseEntry but the latter is used in api responses
+// and the former is used at the DAO level so it seems better to keep them separate
+@Lenses final case class AccessPolicyWithMembership(policyName: AccessPolicyName, membership: AccessPolicyMembership, email: WorkbenchEmail)
```

Comment:
Out of context, your decision here is fine by me.  In context though, we now have 5 (not counting AccessPolicyDescendantPermissions which is a different thing) different representations of an AccessPolicy.  It's confusing and if we can avoid adding more, that's my preference.  But if it's a can of worms, then maybe leave this as is and at least create a tech debt ticket to come back and try to harmonize all these representations and see if we can simplify.